run-build.sh

```diff
@@ -21,11 +21,12 @@ cd $NETLIFY_REPO_DIR
 : ${NODE_VERSION="12.18.0"}
 : ${RUBY_VERSION="2.7.1"}
 : ${YARN_VERSION="1.22.4"}
+: ${PNPM_VERSION="1.22.4"}
```

Comment:
Should that be 7.13.4? Not that it matters, but...

Reply:
you are right – stupid typo but does not matter (for sake of correctness I will update it)